css-positioning/absolute.css

```diff
@@ -0,0 +1,38 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  z-index: 1;
```

Comment:
z-index is not necessary here

Reply:
Haha, thank's tried that first and forgot to remove that one.